readme.md

```diff
@@ -50,9 +50,8 @@ Next, to build from source you will need:
     other distros as well). 
 
 Then, if you are on a \*nix system or Windows, the following steps should compile
-Nim from source using ``gcc``, ``git`` and the ``koch`` build tool (in the place
-of ``sh build.sh`` you should substitute ``build.bat`` on x86 Windows or
-``build64.bat`` on x86_64 Windows):
+Nim from source using ``gcc``, ``git`` and the ``koch`` build tool (for Windows,
+see adjustments needed in `build_all.sh`).
```

Comment:
I think Windows users don't enjoy looking into shell scripts and feel like second class citizens.

Reply:
done: see latest commit